tests/migration/online_migration/yast2_migration.pm

```diff
@@ -220,10 +220,13 @@ sub run {
             send_key 'alt-t';
         }
     }
-    assert_screen ['yast2-migration-installupdate', 'yast2-migration-proposal'], 700;
-    if (match_has_tag 'yast2-migration-installupdate') {
+    if (check_screen('yast2-migration-installupdate', 700)) {
```

Comment:
Why don't we use assert_screen here?

Reply:
Because for some cases, there's no yast2-migration-installupte.

Reply:
Got it, then no problem. can you please add some comments there? then as a new learner, I am a little confusing why you check this screen.

Reply:
ok, no problem and thanks.

Reply:
Done with adding a comment here, thanks.